### PR TITLE
Allow basic search by romaji using title_kana

### PR DESCRIPTION
**Pull request by starrodkirby86**
# Changes
- Import [wanakana](https://wanakana.com/) using unpkg to convert kana into romaji
- Add a `title_romaji` field using a song's `title_kana`
- Add `title_romaji` as a search parameter

# Preview
image

# Shortcomings
- While the inclusion of `title_kana` may seem like a blessing by SEGA to use as a simple means to make romaji, the actual field has its own drawbacks. The primary one is that all dakutens and small kana are normalized (e.g. だから becomes タカラ, じゅぶん becomes シユフン). This does not make searching for certain songs very fun. Everything is also one word, but that's not really a surprise given how Japanese works.
- Wanakana doesn't support kanji conversion (it's lightweight after all!). There are other libraries that do support this like [kuroshiro](https://github.com/hexenq/kuroshiro), but they require additional dependencies that can get easily addressed by turning this into a node project. This may be something for you to consider, but I'm not sure whether Github Pages lets you cleanly approach this. ~~I guess there's always Surge, though...~~

### CODE REVIEW

**Comment by starrodkirby86** [comment]
~~don't mind the force push i just saw an extra line break and i just wanted that killed off~~

File: index.html [hunk 1]
```diff
@@ -13,6 +13,8 @@
 		<meta property="og:url" content="https://gra0007.github.io/maimai-song-list/">
 		<meta name="twitter:card" content="summary_large_image">
 
+		<script src="https://unpkg.com/wanakana"></script>
```
Comment (GRA0007) [inline index.html]: Can you move this to the body before the other script? That way the page can load first.

**Review by GRA0007 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (starrodkirby86) [reply index.html]: oh whoops totally didn't see that other script there, hhehehhhHEHAHhehehHEH

Alright done

**Review by GRA0007 — APPROVED** [review]
Thanks Kirby! Great contribution ❤️

**Pull request merged**